test/test.js

```diff
@@ -132,3 +132,18 @@ sjcl.codec.hex = sjcl.codec.hex ||
     return sjcl.bitArray.clamp(out, len*4);
   }
 };
+
+// CCM ArrayBuffer implementation test relies on sjcl.codec.arrayBuffer.toBuffer
+sjcl.codec.arrayBuffer = sjcl.codec.arrayBuffer ||
```

Comment:
Yes, I agree. I was just trying to see what is the problem. I have removed it now.